roc.yml

```diff
@@ -0,0 +1,69 @@
+--- 
+GENERATOR: 
+  PackageName: roc 
+  PackageDescription: "Package roc provides Go bindings for libroc"
+  PackageLicense: ""
+  Includes:
+    - ../include/roc/address.h
+    - ../include/roc/config.h
+    - ../include/roc/context.h
+    - ../include/roc/frame.h
+    - ../include/roc/log.h
+    - ../include/roc/platform.h
+    - ../include/roc/receiver.h
+    - ../include/roc/sender.h
+  FlagGroups:
+    - {name: "CFLAGS", traits: ["linux"], flags: [
+        "-I\"../include\"",
+    ]}
+    - {name: "LDFLAGS", traits: ["linux"], flags: [
+      "-lroc",
+    ]} 
```

Comment:
Could you explain why do we need `traits: ["linux"]` here? AFAIK cgo always uses gcc or gcc-like compilers so these options should work everywhere?

Reply:
These are platform dependent, even for FreeBSD/Ubuntu the set of libraries might have different names in it. Not sure if true for roc, though.

Reply:
I removed them, so it's now on you @gavv that -lroc will be packaged with the same name on all platforms :)